src/pages/test-report/components/driving-fault-summary/__tests__/driving-fault-summary.spec.ts

```diff
@@ -71,4 +71,26 @@ describe('DrivingFaultSummary', () => {
       expect(summaryCount.textContent).toBe('3');
     });
   });
+
+  describe('driverTypeSwitch()', () => {
+    it('should return R when a category equals EUAM1,', () => {
+      const driverType = component.driverTypeSwitch(TestCategory.EUAM1);
+      expect(driverType).toEqual('R');
+    });
+
+    it('should return R when a category equals EUAM2', () => {
+      const driverType = component.driverTypeSwitch(TestCategory.EUAM2);
+      expect(driverType).toEqual('R');
+    });
+
+    it('should return R when a category equals B', () => {
```

Comment:
should be - 'should return D when a category equals B'

Reply:
corrected